docs/src/main/paradox/_template/page.st

```diff
@@ -71,10 +71,10 @@
 
             <div class="page-content row">
               <div class="small-12 large-9 column" id="docs">
-                $page.content$
-                
+$page.content$
```

Comment:
FYI: `$! comment !$` can be used to add comments on the ST4 level which won't be added to the output.